websites/F/Flat/presence.ts

```diff
@@ -0,0 +1,82 @@
+const presence = new Presence({
+	clientId: "1021848171946704906",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/pFfJvOZ.png",
+		},
+		{ pathname } = location;
+	if (pathname === "/") presenceData.details = "Viewing the homepage";
+	if (pathname === "/my-library")
+		presenceData.details = "Viewing their library";
+	if (pathname.startsWith("/collection/")) {
+		presenceData.details = "Viewing a collection:";
```

Comment:
Ideally shouldn't have colons (`:`)
```suggestion
		presenceData.details = "Viewing a collection";
```